zproject/settings.py

```diff
@@ -198,9 +198,12 @@ def get_secret(key):
                     # than a separate app.
                     'EXTRA_INSTALLED_APPS': ['analytics'],
                     'DEFAULT_NEW_REALM_STREAMS': {
-                        "social": {"description": "For socializing", "invite_only": False},
-                        "general": {"description": "For general stuff", "invite_only": False},
-                        "zulip": {"description": "For zulip stuff", "invite_only": False}
+                        "social": {"description": "For socializing", "invite_only": False,
```

Comment:
I think we should not add these (custom_welcome_message) here, even as an intermediate step. Changes to the settings file are particularly messy, due to ensuring that the migration from the old setting file to the new works well.